### PR TITLE
Perf cluster setup updates

### PR DESCRIPTION
**Pull request by ldemailly**
Comments about how to run it / note about needing correct istioctl and
matching istio-auth.yaml

```release-note
NONE
```

### CODE REVIEW

File: tools/cache_buster.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+---
+apiVersion: "config.istio.io/v1alpha2"
+kind: rule
+metadata:
+  name: mixercachebuster
+spec:
+  # one direction 1->2 won't be cached, 2->1 will use the cache
+  # TODO: parametrize the namespace of find a way to get short names to work:
+  # TODO: this appears to always bust the cache, even if dest is echosrv1 !
+  match: destination.service == "echosrv1.istio.svc.cluster.local" && request.headers["x-request-id"] == "foo"
```
Comment (ldemailly) [inline tools/cache_buster.yaml]: cc @mandarjog

**Comment by ldemailly** [comment]
@mandarjog @ozevren to repro the current issues:

```
export PROJECT=youproject
source tools/setup_perf_cluster.sh
  install_istio_svc
  install_istio_ingress_rules
  install_istio_cache_busting_rule
  get_istio_ingress_ip
```

File: tools/cache_buster.yaml (hunk 1 above)
Comment (douglas-reid) [reply tools/cache_buster.yaml]: this shouldn't break the cache moving forward, correct?  have we verified with latest proxy (i believe there was an issue where we were still using a weeks-old build) ?

**Pull request merged**